### DISCUSSION TITLE
SD-XL Turbo, cascade, ...

### DISCUSSION THREAD

**Discussion by darth2602** [TheLastBen/fast-stable-diffusion#2746] [Q&A]
Bonjour
est ce que A1111 est compatible avec les nouveaux Modèles de SDXL  (Turbo, Cascade,...) 

Il y a deux modèles pour Cascade, B et C est ce qu'il faut en utiliser un seul ou est ce que B est le principal et C le refiner ?

**Comment by TheLastBen** [comment]
Salut, 
Utilise ComfyUI pour Cascade, et pour lightning, LoRA suffit, et ca marche avec A1111, beaucoup mieux que turbo

**Comment by darth2602** [comment]
Salut 
je n'utilise pas Comfy (beaucoup trop compliqué pour le même résultat), j'utilise A111 avec Google Colab et donc SD Web UI .... 
Si j'essaye avec les modèle B ou C safetensor ça va marcher dans Web UI ? (c'est quoi la différence entre B et C  ?  je peux utiliser seulement 
B ou seulement C ?    est ce que je peux utiliser le SDXL refiner 1.0 ou je n'en ai pas besoin  ?

**Reply by TheLastBen** [reply]
Pas de support pour cascade dans A1111 pour l'instant
